stable/cachet/values.yaml

```
@@ -24,6 +24,9 @@ ingress:
   host: chart-example.local
   tls: []
   path: /
+  # -- Set ingressClass for Ingress when needed
+  ingressClassName: ingressClass
```

I think this should be commented by default otherwise `ingressClass` will be used?

```suggestion
  # ingressClassName: ingressClass
```

---

yes, you are right. updated it.